app/src/main/java/com/github/mik629/android/fundamentals/ui/AppActivity.kt

```diff
@@ -3,16 +3,12 @@ package com.github.mik629.android.fundamentals.ui
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
 import com.github.mik629.android.fundamentals.R
-import com.github.mik629.android.fundamentals.databinding.ActivityMainBinding
 import com.github.mik629.android.fundamentals.ui.movieslist.FragmentMoviesList
 
-class MainActivity : AppCompatActivity() {
-    private lateinit var binding: ActivityMainBinding
-
+class AppActivity : AppCompatActivity() {
```

Comment:
class AppActivity : AppCompatActivity(R.layout.activity_main)

and remove setContentView(R.layout.activity_main)